docker/alpine-dind/toscana-dind-entrypoint.sh

```diff
@@ -1,3 +1,6 @@
 #!/bin/bash
 dockerd &
-java -jar server.jar
+logfile=/var/log/toscana.log
```

Comment:
I would consider this a Environment Variable (in the dockerfile) `ENV LOGFILE=/var/log/toscana.log` To allow external customisation. 

This would have to be added in the dockerfile for the jenkins deployment and the one used by dockerhub